website/templates/_activity.html

```diff
@@ -4,9 +4,13 @@
         {% if activity.user.userprofile.avatar %}
             <img src="{{ activity.user.userprofile.avatar }}" width="100" class="img-responsive img-rounded">
         {% elif activity.user.socialaccount_set.all.0.get_avatar_url %}
-            <img src="{{ activity.user.socialaccount_set.all.0.get_avatar_url }}" width="100" class="img-responsive img-rounded">
-        {% else %}
+            <img src="{{ activity.user.socialaccount_set.all.0.get_avatar_url }}" width="100"
+                 class="img-responsive img-rounded">
+        {% elif activity.user.email %}
             <img src="{% gravatar_url activity.user.email 100 %}" width="100" class="img-responsive img-rounded">
+        {% else %}
+            <img src="https://heightstitle.com/wp-content/uploads/2014/12/user-dummy-pic.png" width="100"
```

Comment:
@spiderxm can you please move this image into static?  The first time I tried to load it, I got a 404, so will be good to have it hosted with us.

Reply:
ok sure